### DISCUSSION TITLE
String parameter default value

### DISCUSSION THREAD

**Discussion by MichaelStarks** [nasa/fprime#1733] [Q&A] [answered]
Hello,

I am trying to set the default value of a string parameter but I am running into a build error. I have the following line in my component's fpp file:

`param SCHEDULE_STRING: string size 11 default "0 * * * * ?" id 2 \
      set opcode 0x0003 \
      save opcode 0x0004
`

The auto-generated XML file puts an additional set of double quotes around the default value:

`<parameter id="0x2" set_opcode="0x3" save_opcode="0x4" name="SCHEDULE_STRING" data_type="string" size="11" default=""0 * * * * ?""/>
`

Should I format my string differently to prevent the compilation error?

**Comment by bocchino** [comment, marked as answer]
This is a bug in fpp-to-xml. I created an issue here to track it: https://github.com/fprime-community/fpp/issues/178.
